modules/react/common/lib/theming/useTheme.ts

```diff
@@ -45,7 +44,7 @@ export function useTheme(theme?: PartialEmotionCanvasTheme): EmotionCanvasTheme
     // Context not supported or invalid (probably called from within a class component)
   }
 
-  const windowTheme = typeof window !== 'undefined' && get(window, 'window.workday.canvas.theme');
+  const windowTheme = typeof window !== 'undefined' && (window as any)?.workday?.canvas?.theme;
```

Comment:
The `get` avoided the `window as any`. This is typechecked elsewhere. Optional chaining will prevent errors at runtime.